angularjs-portal-home/src/main/webapp/my-app/search/controllers.js

```diff
@@ -134,9 +141,11 @@ define(['angular', 'portal/search/controllers', 'my-app/marketplace/controllers'
       };
       init();
       if(googleCustomSearchService.googleSearchEnabled()){
+        $scope.googleSearchLoading = true;
```

Comment:
Move these loading flags into the init function right before the $http.get. Having it here takes it out of context and its confusing.